pkg/reconciler/storage/storage.go

```diff
@@ -280,8 +280,10 @@ func (c *Reconciler) reconcileNotification(ctx context.Context, storage *v1alpha
 	}
 
 	customAttributes := make(map[string]string)
-	for k, v := range storage.Spec.CustomAttributes {
-		customAttributes[k] = v
+	if storage.Spec.CloudEventOverrides != nil {
```

Comment:
the cloudEventOverrides should propagate down into the PullSubscription, not added to the customAttributes

Reply:
I don't understand the tradeoffs, so I made the minimum change to match the desired interface.